packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -43,6 +43,7 @@ describe('useMachine hook', () => {
       },
       loading: {
         invoke: {
+          id: 'fetchData',
```

Comment:
q: was there a reason behind adding this here? couldn't this rely on the auto-generated id? I'm just wondering if this had to be added here or if it could use the auto-generated id to assess the scope of the change

Reply:
It _can_ use the autogenerated ID, but the test is specifically testing:

```js
//          vvvvvvvvv
doneInvoke('fetchData', 'persisted data')
```

So I wanted to keep that part of the test the same.

Reply:
👌